src/core.js

```diff
@@ -90,14 +90,29 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     registerAsRootInstance(this);
   }
 
-  keyStateStartObserving();
+  this.rootElement = rootElement;
+  /**
+   * The nearest document over container.
+   *
+   * @private
+   * @type {Document}
+   */
+  this.rootDocument = rootElement.ownerDocument;
```

Comment:
Maybe add here comment that it's a reference to the same element as in the Walkontable. Any `TODO` comment informing that it should be changed.